lib/tapioca/compilers/requires_compiler.rb

```diff
@@ -49,13 +50,40 @@ def collect_requires(file_path)
         end.compact
       end
 
-      sig { params(config: Spoom::Sorbet::Config, file: String).returns(T::Boolean) }
-      def file_ignored_by_sorbet?(config, file)
-        config.ignore.any? do |path|
-          Regexp.new(Regexp.escape(path)) =~ file
+      sig { params(config: Spoom::Sorbet::Config, file_path: Pathname).returns(T::Boolean) }
+      def file_ignored_by_sorbet?(config, file_path)
```

Comment:
Should we move this service to `Spoom`? Seems useful.

Reply:
Yeah, I was thinking the same thing. Let's do it in a future refactor, and we can add tests around it as well.